### ISSUE TITLE
Date picker a11y issues

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1171]
Screen reader test, investigate and fix the various issue with pikaday:

![image](https://user-images.githubusercontent.com/756393/77053907-f39fd200-69c6-11ea-9078-b571c8c68a4e.png)

**Comment by Stanton** [comment]
Pikaday concerns me, the last repo activity was 17 months ago and I'm not sure we should support another fork of our own like we're doing with Select-woo...

https://www.webaxe.org/accessible-date-pickers/

**Comment by jamesjacobs** [comment]
### Issues found

**SiteImprove**
- year `<select>` has no label
- month `<select>` has no label
- Missing required WAI-ARIA states or properties. Refers to use of `role="heading"` on `pika-title-XX`.
- HTML is used to format content. Refers to date table.
- When more than 28 inputs with date pickers are present, pikaday starts creating duplicate IDs.

Errors are duplicated for each date input.

**Screen reader**
SR experience is broken. When an input with a date picker is selected and the pikaday calendar shown and SR announces nonsense. Same happens when arrow keys used. Original input label not announced. See screenshot below for Mac Vo output when main input is selected:

![Screenshot 2020-04-01 at 11 48 20](https://user-images.githubusercontent.com/756393/78129660-d67bf200-740f-11ea-9f3f-7a58e7361267.png)

Going to explore accessible alternatives to pikaday.

**Comment by jamesjacobs** [comment]
List of potentially accessible date pickers for investigation:
- https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html
- https://dequeuniversity.com/library/aria/sf-date-picker
- https://backpack.github.io/components/datepicker?platform=web
- http://airbnb.io/react-dates/?path=/story/singledatepicker-sdp--as-part-of-a-form
-
